frontend/app/templates/components/badge-design.hbs

```diff
@@ -0,0 +1,35 @@
+{{!--
+<div class="ui grid">
+  <div class="six wide column">
+
+  </div>
+  <div class="six wide column">
+    <h3>Features</h3>
+  </div>
+</div> --}}
+<div class="wrapper">
+          <br>
```

Comment:
instead of adding `<br>` tags we can have margins setup on the associated `css` for the given `html` code.
Can you please make the changes so that I can merge this? :smiley: